docs/website/docs/plus/features/project/source-configuration.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Configuration of Source
+description: Initialize ore sources in yaml file
+---
+
+# Configuration of Sources
+
+<img src="https://storage.googleapis.com/dlt-blog-images/plus/dlt_plus_projects.png" width="500"/>
+
+
+The `dlt.yml` file is used to declaratively configure data sources for use with the dlt CLI. This setup provides a flexible way to manage data extraction for the core sources REST APIs, SQL databases and file systems using YAML syntax.
+
+Credential placeholders for the defined sources are automatically generated in `.dlt/secrets.toml`. Alternatively, credentials may also be provided directly within `dlt.yml`.
+
+
+## REST API
+
+The built-in `rest_api` type enables configuration of REST-based integrations. Multiple endpoints can be defined under a single source.
+
+
+```yaml
+sources:
+  pokemon_api:
+    type: rest_api
+    client:
+      base_url: https://pokeapi.co/api/v2/
+      paginator: auto
+    resource_defaults:
+      primary_key: name
+    resources:
+      - pokemon
+      - berry
+      - 
+        name: encounter_conditions
+        endpoint:
+          path: encounter-condition
+          params:
+            offset:
+              type: incremental
+              cursor_path: name
+        write_disposition: append
+```
+
+* `type: rest_api`: Specifies the use of the built-in REST API source.
+* `client.base_url`: Sets the root URL for all API requests.
+* `paginator: auto`: Enables automatic detection and handling of pagination.
+* `resource_defaults`: Contains the default values to configure the dlt resources. This configuration is applied to all resources unless overridden by the resource-specific configuration.
+* Each item in `resources`defines an endpoint to extract. Simple entries like `pokemon`and `berry` will fetch from `/pokemon` and `/berry`, respectively.
+* The `encounter-condition` resource uses an advanced configuration:
+  * `path`: Point to the `/encounter-condition`endpoint.
+  * `params.offset`: Enables incremental loading using the `name` field as the cursor.
+  * `write_disposition: append`: Ensures new data is appended rather than overwriting previous loads.
```

Comment:
Maybe let's use any other write disposition, because `append` is the default one, so it might be confusing why we're specifying it